spec/attr_extras/attr_initialize_spec.rb

```diff
@@ -33,6 +33,27 @@ def self.name
     example.instance_variable_get("@baz").must_equal "Baz"
   end
 
+  it "supports default values for hash values" do
+    klass = Class.new do
+      attr_initialize :foo, [:bar, :baz => "WOW", :victor => "arias"]
+    end
+
+    example = klass.new("Foo", :bar => "Bar", victor: "garcia")
+    example.instance_variable_get("@foo").must_equal "Foo"
+    example.instance_variable_get("@bar").must_equal "Bar"
+    example.instance_variable_get("@victor").must_equal "garcia"
+    example.instance_variable_get("@baz").must_equal "WOW"
+  end
+
+  it "accepts a hash as a positional argument" do
```

Comment:
Maybe a comment saying `# Regression` so it reads clearer – this isn't some interesting piece of behavior, just bug avoidance :)